### PR TITLE
mgr/orch: 'ceph orchestrator ...' -> 'ceph orch ...'

### PR DESCRIPTION
**Pull request by liewegas**
Less typing for all!

### CODE REVIEW

File: doc/mgr/orchestrator_modules.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ ceph-mgr orchestrator modules
 
 In this context, *orchestrator* refers to some external service that
 provides the ability to discover devices and create Ceph services.  This
-includes external projects such as ceph-ansible, DeepSea, and Rook.
+includes external projects such as DeepSea and Rook.
```
Comment (mgfritch) [inline doc/mgr/orchestrator_modules.rst]:
```suggestion
includes external projects such Rook.
```

**Comment by leseb** [comment]
It would have been nice to keep backward compatibility!

**Comment by liewegas** [comment]
we can kludge it into the CLI. I didn't think there were any users of this API though, since this didn't do much in nautilus?

**Comment by sebastian-philipp** [comment]
This caused https://github.com/rook/rook/issues/4517

**Comment by sebastian-philipp** [comment]
@leseb do you need a `ceph orchestrator set backend` kludge for Rook?

**Comment by leseb** [comment]
@sebastian-philipp this was fixed already in https://github.com/rook/rook/pull/4977

**Comment by sebastian-philipp** [comment]
sigh. sorry for the inconvenience. We really need to integrate ceph's qa and rook better.

**Pull request merged**